generate.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-while getopts g:t:m:u:w: option
+while getopts g:t:d:m:u:w: option
 do
  case "${option}"
  in
  g) GITURL=${OPTARG};;
  t) export DOCTHEME=${OPTARG};;
+ d) export DEBUG=${OPTARG};;
```

Comment:
Why do we export it here? don't you think setting this enviornment variable of the system would create hinderance in concurrent execution of Documentation generation?